docs/backup/new-backup-strategy.md

```diff
@@ -0,0 +1,164 @@
+# Kyma Backup Strategy
+
+User load on a Kyma cluster typically consists of various Kubernetes objects and volumes. Kyma relies on the backing cloud provider for periodic backups of the Kubernetes objects. That's why it does not require the user to do any manual settings to take backups.
+
+For example, Gardener uses etcd as the Kubernetes' backing store for all cluster data. That is, all Kubernetes objects are stored on etcd. Gardener has perodic jobs to take major and minor backups of etcd database. A major backup including all the resources takes place every hour, and each minor backups including only the changes inbetween take place every five minutes. In case of a problem on etcd database, Gardener automatically restores the Kubernetes cluster using the latest backup.
+
+However, volumes are typically not a part of these backups. That's why Kyma encourages users to take periodic backups of their volumes. This can be done using the new Kubernetes API `VolumeSnapshot` that is explained below.
+
+## On-Demand Volume Snapshots
+
+Kubernetes provides a new API resource called VolumeSnapshot that can be used to take the snapshot of a volume on Kubernetes. A snapshot can be used either to provision a new volume (pre-populated with the snapshot data) or to restore the existing volume to a previous state (represented by the snapshot).
+
+VolumeSnapshot support is only available for CSI drivers. However, not all the CSI drivers support the volume snapshot functionality. You can find a list of all the drivers with the supported functionalities [here](https://kubernetes-csi.github.io/docs/drivers.html).
+
+An example VolumeSnapshot created to take a snapshot from the PVC named `pvc-to-backup`:
```

Comment:
Make it more clear that now the tutorial part starts.
Tell a story. So assumed you have a PVC like that. Then create a Snapshot. Now assume your PVC is corrupt, does not work anymore and you want to re-create it, then...